reference/3.0/Microsoft.PowerShell.Core/About/about_Types.ps1xml.md

```diff
@@ -46,109 +44,117 @@ For example, Windows PowerShell adds a DateTime property to all
 System.DateTime objects, such as the ones that the Get-Date cmdlet
 returns.
 
+```powershell
 PS C:> (Get-Date).DateTime
 Sunday, January 29, 2012 9:43:57 AM
+```
 
-You won't find the DateTime property in the description of the System.DateTime
-structure (http://msdn.microsoft.com/library/system.datetime.aspx),
+You won't find the DateTime property in the description of the [`System.DateTime`
+structure](http://msdn.microsoft.com/library/system.datetime.aspx),
 because Windows PowerShell adds the property and it is visible only in
 Windows PowerShell.
 
 To add the DateTime property to all Windows PowerShell sessions, Windows PowerShell
 defines the DateTime property in the Types.ps1xml file in the Windows PowerShell
-installation directory ($pshome).
+installation directory (`$PSHOME`).
 
 Adding Extended Type Data to Windows PowerShell.
 
 There are three sources of extended type data in Windows PowerShell sessions.
 
---  The Types.ps1xml files in the Windows PowerShell installation directory
+-  The Types.ps1xml files in the Windows PowerShell installation directory
 are loaded automatically into every Windows PowerShell session.
-
---  The Types.ps1xml files that modules export are loaded when the module
+-  The Types.ps1xml files that modules export are loaded when the module
 is imported into the current session.
-
---  Extended type data that is defined by using the Update-TypeData cmdlet
+-  Extended type data that is defined by using the Update-TypeData cmdlet
 is added only to the current session. It is not saved in a file.
 
 In the session, the extended type data from the three sources is applied
 to objects in the same way and is available on all objects of the specified
 types.
 
-The TypeData Cmdlets
+## The TypeData Cmdlets
 
 The following TypeData cmdlets are included in the Microsoft.PowerShell.Utility
 module in Windows PowerShell 3.0 and later versions of Windows PowerShell.
 
-Get-TypeData:     Gets extended type data in the current session.
-Update-TypeData:  Reloads Types.ps1xml files. Adds extended type
-data to the current session.
-Remove-TypeData:  Removes extended type data from the current
-session.
+|              |                                                 |
```

Comment:
oh, I missed this - without a header, the table renders strangely in GitHub. I don't think that should block this checkin though. If it shows up ugly on the web too, we can always tweak it further.